src/docfx/build/Build.cs

```diff
@@ -105,7 +106,8 @@ async Task BuildOneFile(Document file, Action<Document> buildChild)
 
                 bool ShouldBuildFile(Document file)
                 {
-                    return file.ContentType != ContentType.Unknown && filesBuilder.TryAdd(file);
+                    return file.ContentType != ContentType.Unknown && filesBuilder.TryAdd(file) &&
+                        (docset.FallbackDocset == null/*source docset*/ || file.Docset.FallbackDocset != null /*loc file*/);
```

Comment:
In which case, the file to be built is neither from source docset nor a loc file? And we want to exclude it from build scope

Reply:
？

---
In reply to: [227236163](https://github.com/dotnet/docfx/pull/3519#discussion_r227236163) [](ancestors = 227236163)